src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```diff
@@ -91,7 +118,13 @@ const SearchTabPanel = () => {
             }
         >
             <Box className={"search-tab-container"}>
-                <QueryInputBox/>
+                {isKqlFilteringEnabled ?
+                    <>
+                        <FilterInputBox/>
+                        <div className={"divider"}/>
+                        <QueryInputBox/>
+                    </> :
```

Comment:
i agree with the rabbit's comment at https://github.com/y-scope/yscope-log-viewer/pull/371/files#r2317360664 about removing the divider component. add a wrapper that wrap both the boxes with a gap in between better matches the semantics and might avoid any overflow (in the y-direction) issues.

```suggestion
                    <div className={"query-input-boxes-container"}>
                        <FilterInputBox/>
                        <QueryInputBox/>
                    </div> :
```

(subjectively, 8px seems better than 10px
```
.query-input-boxes-container {
    display: flex;
    flex-direction: column;
    gap: 8px; 
}
```